composer.json

```
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
```

```suggestion
        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || ^3.0@beta",
```
the new things are only in the 3.x branch of the bundle. i only did a beta release of it so far. lets install that to see if it works.